documentation/modules/cruise-control/proc-generating-optimization-proposals.adoc

```diff
@@ -125,7 +125,30 @@ Status:
 ----
 +
 The properties in the `Optimization Result` section describe the pending cluster rebalance operation. 
-For descriptions of each property, see xref:contents-optimization-proposals[Contents of optimization proposals]. 
+For descriptions of each property, see xref:contents-optimization-proposals[Contents of optimization proposals].
+
+.Insufficient CPU capacity
+
+If a Kafka cluster is overloaded in terms of CPU utilization, you might see an insufficient CPU capacity error in the `KafkaRebalance` status. 
+This might be because of excluded topics configuration (`excludedTopics`). 
```

Comment:
Adding comments from an "outdated" thread here for visibility:

I felt that this sentence makes it sound as if excludedTopics could be the cause of the brokers being overloaded. From what I understand, excludedTopics does not affect the load on the brokers, it affects what partitions are allowed to be moved?

I thought the mention of excludeTopics was to warn users who are expecting excludeTopics to affect the load that excludeTopics has no affect on the load. Was that the intention or am I misunderstanding something?

Reply:
I agree it is not clear enough.

AFAIK, when you set an `excludeTopics` regular expression, all topic names matching that expression are excluded from partition replica/leader movements in the optimization proposal. For that reason you, may never hit the CPU goal. Obviously, that would be a misconfiguration.